questions/css-box-model.md

```diff
@@ -0,0 +1,31 @@
+### Your question goes here. Keep it straightforward.
+
+Describe the layout of the CSS Box Model and briefly describe each component.
+
+#### Answer
+
+<!-- Your answer goes here. -->
+
+*Content*: The inner-most part of the box filled with text or an image or some type of content</br>
+
+*Padding*: The transparent area surrounding the content</br>
+
+*Border*: The area surrounding the padding (if any) and content</br>
+
+*Margin*: The transparent outer-most layer that surrounds the border
+
+#### Good to hear
+
+<!-- Whenever possible, include the short list of bullet points that sum up the answer. -->
+This is a very common question to be asked during a front-end interview and something that you should be very comfortable with.
```

Comment:
This is supposed to be a list(preceed the line with a `-` or `*` followed by a space) listing the benefits of the answer and it's relivance

> - This is a very common question asked in almost any front-end interview
> * Shows an understanding of the spacing an decoration model employed by the DOM and it's elements

The top was a dash, the bottom an asterisk to show that they appear no different

Reply:
Thank you!  Admittedly, I am not entirely used to markdown but I will implement that now.